packages/ckeditor5-table/docs/_snippets/features/table-default-properties.html

```diff
@@ -20,16 +20,32 @@
 
 <div class="table-default-properties-container">
     <div id="snippet-table-default-properties">
+    <p>Before people could leave the cradle of humanity and enter the cosmic void, tests were required. A number of animals risked and sometimes lost their lives to open the gates for their owners to be able to reach the stars.</p>
+    <p>The animal spaceflights started in the late 1940s. German A4 rockets were used for the initial launches, later on each country developed their own carriers. Even though these simian, canine and rodent space farers were the most important test subjects during the first stage of the space race, before launching actual human cosmonauts, the animal spaceflight is very much a thing even today. In 2018 a rodent crew arrived at International Space Station!</p>
```

Comment:
```suggestion
    <p>The animal spaceflights started in the late 1940s. German A4 rockets were used for the initial launches, later on each country developed their own carriers. Even though these simian, canine and rodent space farers were the most important test subjects during the first stage of the space race, before launching actual human astronauts, the animal spaceflight is very much a thing even today. In 2018 a rodent crew arrived at International Space Station!</p>
```

Reply:
Both are correct, by the way. Andcosmonaut is universally used for East, so Laika was a cosmonaut ;)